yarn.lock

```diff
@@ -865,6 +1008,24 @@ heroku-client@^2.4.0:
     q "^1.2.0"
     tunnel-agent "^0.4.0"
 
+heroku-container-registry@4.3.0:
+  version "4.3.0"
+  resolved "http://cli-npm.heroku.com/heroku-container-registry/-/heroku-container-registry-4.3.0/433e6e311ac8c289b3f1845e9b1f633483f3c390.tgz#433e6e311ac8c289b3f1845e9b1f633483f3c390"
+  dependencies:
+    camelcase "1.0.2"
```

Comment:
They're all in [the package.json file](https://github.com/heroku/heroku-container-registry/blob/91d74da32e522d58504f76a817820aeb311823d3/package.json#L20-L33).